package OpenSolid;

message SerializedMatrixXd {
    required int32 rows = 1;
    required int32 cols = 2;
    repeated double data = 3 [packed=true];
}

message SerializedMatrixXI {
    required int32 rows = 1;
    required int32 cols = 2;
    repeated bytes data = 3;
}
